tests/test_liquidity_pool.py

```diff
@@ -219,7 +231,127 @@ def test_liquidity_pool(w3, contract, DAI_token, USDC_token, price_oracle, asser
     # Can add liquidity again after all liquidity is divested
     contract.addLiquidity(DAI_token.address, DAI_ADDED, DEADLINE, transact={'from': user1})
 
+def test_liquidity_gusd(w3, contract, DAI_token, GUSD_token, price_oracle, assert_fail):
```

Comment:
It seems ```test_liquidity_gusd``` is quite similar to ```test_liquidity_pool``` (but uses GUSD instead of USDT). What is an idea of having both them? Is there anything that should be tested especially for USDT (6 decimals)? Or maybe it is possible to adapt tests we already had to use GUSD instead of USDT (and maybe to add few additional asserts if necessary) to keep a full cover with a lower number of lines?